packages/react/package.json

```diff
@@ -30,6 +30,7 @@
         "@tanstack/react-query-persist-client": "^4.24.9",
         "@uniswap/token-lists": "1.0.0-beta.31",
         "class-variance-authority": "^0.4.0",
+        "react-error-boundary": "^4.0.2",
```

Comment:
Wondering if we could use an error boundary directly. It's an official React feature and very simple to implement, I don't think we necessarily need a new lib.

https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary